plugins/rollbar-backend/src/api/RollbarApi.ts

```diff
@@ -0,0 +1,216 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import axios from 'axios';
+import { Logger } from 'winston';
+import camelcaseKeys from 'camelcase-keys';
+import { buildQuery } from '../util';
+import {
+  RollbarItemCount,
+  RollbarItemsResponse,
+  RollbarProject,
+  RollbarProjectAccessToken,
+  RollbarTopActiveItem,
+} from './types';
+
+const baseUrl = 'https://api.rollbar.com/api/1';
+
+const buildUrl = (url: string) => `${baseUrl}${url}`;
+
+export class RollbarApi {
+  private projectMap: ProjectMetadataMap | undefined;
+
+  constructor(
+    private readonly logger: Logger,
+    private readonly accessToken?: string,
+  ) {
+    this.accessToken = accessToken || getRollbarToken(logger);
```

Comment:
Yeah, I'm not particular happy with this as is either. I'm going to move this back to at least the router. However it will be similar except contained within the `createRouter` method. The other option is to push it up the stack even further to the backend where it calls `createRouter`.

I also thought about leveraging the configuration mechanism as well, but it didn't feel right. This might be an area to think about standardizing on a convention of how environment variables should be loaded and consumed by plugins.